release/Makefile

```diff
@@ -108,8 +108,7 @@ host: $(OUTDIR)/opam-full-$(VERSION).tar.gz build/$(HOST).env
 	    MAKE=$(MAKE) \
 	    $(EXPORTS_$(HOST_OS)); \
 	  cd build/opam-full-$(VERSION) && \
-	  ./configure && \
-	  { $(MAKE) lib-ext || ./configure --with-vendored-deps; } && \
+	  ./configure --with-vendored-deps && \
```

Comment:
This is still required to make releases of the 2.1 branch

Reply:
There is also another thing that is 2.1 related, sha dependency. Should we have conditional commands according 2.1/2.2 ?

Reply:
What is sha dependency?

Reply:
This [one](https://github.com/ocaml/opam/blob/master/release/Makefile#L58)